<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LadeSeite</class>
 <widget class="QWidget" name="LadeSeite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Futterkarre 1.0 - Futter laden</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fdffe0;</string>
  </property>
  <widget class="QLabel" name="ub_laden">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2E672F;
color: #fdffe0;
</string>
   </property>
   <property name="text">
    <string>Futter laden</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_karre_gewicht">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>671</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic</family>
     <pointsize>170</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_w_bestaetigen">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>580</y>
     <width>300</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2E672F; color: #fdffe0;</string>
   </property>
   <property name="text">
    <string>Beladung best√§tigen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_settings">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>20</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2E672F</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../Futterwagen/Futterkarre/icons/icons.qrc">
     <normaloff>:/icons/Gear.png</normaloff>
     <normalon>:/icons/Gear.png</normalon>:/icons/Gear.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_heu_laden">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>120</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #b0cfa5;
    color: #ffffff;
    border-radius: 15px;
}
QPushButton:hover {
    background-color: #8FBC8F;
    color: #f5f5b5;
}
QPushButton:pressed,
QPushButton:checked {
    background-color: #a30000;  /* Dunkelrot */
    color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>HEU</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_heulage_laden">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>160</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #b0cfa5;
    color: #ffffff;
    border-radius: 15px;
}
QPushButton:hover {
    background-color: #8FBC8F;
    color: #f5f5b5;
}
QPushButton:pressed,
QPushButton:checked {
    background-color: #a30000;  /* Dunkelrot */
    color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>HEULAGE</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_wa_nullen">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>580</y>
     <width>300</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2E672F; color: #fdffe0;</string>
   </property>
   <property name="text">
    <string>Waage Nullen</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_kg">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>450</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>kg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2E672F</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../Futterwagen/Futterkarre/icons/icons.qrc">
     <normaloff>:/icons/Back.png</normaloff>:/icons/Back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../Futterwagen/Futterkarre/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
